src/lt_port_wrap.h

```diff
@@ -92,7 +92,8 @@ lt_ret_t lt_l1_delay_on_int(lt_l2_state_t *s2, uint32_t ms) __attribute__((warn_
 #endif
 
 /**
- * @brief Generate `count` random bytes using host's random number generator. This is wrapper for platform defined function.
+ * @brief Generate `count` random bytes using host's random number generator. This is wrapper for platform defined
```

Comment:
Missing article maybe?
```suggestion
 * @brief Generate `count` random bytes using host's random number generator. This is a wrapper for platform defined
```